README.md

```diff
@@ -86,6 +86,18 @@ cargo run -- \
   --bucket foobar
 ```
 
+If you just need to use the local disk as the backend, use the following bash.
+
+```bash
+# Change this to the output of `openssl rand -hex 32`.
+export RUDOLFS_KEY=282013cfb408da0a27079f515cd76f8793f6565a86e90b97ba36787fe14c6966
```

Comment:
Just so folks don't use this key verbatim out of laziness, I'd put a clearly bogus key here.

Something like this:
```
export RUDOLFS_KEY=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
```
(Same as is done elsewhere in the docs.)

Reply:
solved.